nuxeo-rating-core/src/test/java/org/nuxeo/ecm/rating/TestLikeServiceWithDocs.java

```diff
@@ -399,9 +399,13 @@ protected void createAndLikeDocs(int nb) {
 
     protected String addMiniMessage(NuxeoPrincipal principal, String message, Date publishedDate,
             String contextActivityObject) {
-        Activity activity = new ActivityBuilder().actor(ActivityHelper.createUserActivityObject(principal)).displayActor(
-                ActivityHelper.generateDisplayName(principal)).verb("minimessage").object(message).publishedDate(
-                publishedDate).context(contextActivityObject).build();
+        Activity activity = new ActivityBuilder().actor(ActivityHelper.createUserActivityObject(principal))
+                                                 .displayActor(ActivityHelper.generateDisplayName(principal))
+                                                 .verb("minimessage")
+                                                 .object(message)
+                                                 .publishedDate(publishedDate)
+                                                 .context(contextActivityObject)
+                                                 .build();
```

Comment:
Please void reformattings or cleanups mixed with other code changes, it helps reviews a lot.

Reply:
deformatted
👍